### ISSUE TITLE
Additional content types

### ISSUE THREAD

**Issue by cjallen1** [10up/SketchPress#18]
Do we need to add in the following content types? Would we need these in the future?

- Audio (not overly used, but might be worthwhile)
- Gallery (although this is the same as adding image)
- Heading
- Sub-heading
- List
- Quote
- Code (would we recommend or need this for clients?!)
- Custom HTML (again, would we recommend this?)
- Preformatted
- Pull Quote
- Table
- Verse
- Buttons
- More
- Seperator
- Text Columns

As well as the embeds that can be added, particularly with the Twitter, YouTube and the main ones?

I also think it'd be worthwhile putting together a template for the 'Blocks' tab for these content blocks, where the settings are changed for the block, I'd imagine this is the areas we will be customising for Gutenberg

**Comment by jarahsames** [comment]
@cjallen1 agreed - the original document had core items (and now some core Gutenberg items). Feel free to create any of these items and send a pull request so I can merge it to the master. I will add them on the todo on my end, too!
